include/GafferImage/Median.h

```diff
@@ -56,42 +56,6 @@ class Median : public ImageProcessor
 
 		IE_CORE_DECLARERUNTIMETYPEDEXTENSION( GafferImage::Median, MedianTypeId, ImageProcessor );
```

Comment:
We need to change this so that we're declaring RankFilter as the base class instead of ImageProcessor :

```
IE_CORE_DECLARERUNTIMETYPEDEXTENSION( GafferImage::Median, MedianTypeId, RankFilter );
```

This is the key to getting the metadata to be inherited properly, so we an drop the `commonPlugsMetadata()` thing.